include/erlang_ls.hrl

```diff
@@ -560,6 +562,29 @@
         , more_trigger_character  => string()
         }.
 
+%%------------------------------------------------------------------------------
+%% Code Actions
+%%------------------------------------------------------------------------------
+
+-define(CODE_ACTION_KIND_QUICKFIX, 1).
```

Comment:
Where is the `1` specified in the specs? I can't find it...

Reply:
I think it must have changed in the spec over time, the original value came in two years ago, via https://github.com/erlang-ls/erlang_ls/commit/525229bc70ddcf33d40c7631a460cabb3b2f370b